### PR TITLE
Update WCC Team

### LINKED ISSUE #369: bug: On the meet our team page, Adriana's linkedin is navigating to the wrong linkedin page
### Describe the bug
On the meet our team page, Adriana's linkedin is navigating to the wrong LinkedIn page

### To Reproduce
1. Go to About us page
2. Click on team
3. Scroll down to Adriana Zencke
4. Click on the LinkedIn icon
5. See error

### Expected behaviour
It takes you to https://www.linkedin.com/in/rajaniraog/ instead of Adriana's Linkedin

### Desktop (please complete the following information):
 - Device: [Mac]
 - OS: [Sonoma 14.5]
 - Browser [Google chrome]

### Reproducibility
- [X] The bug is reproducible consistently.
- [ ] The bug is sporadic or occurs randomly.

### Impact
- [X] The bug affects critical functionality.
- [ ] The bug affects minor functionality.
- [ ] The bug is a cosmetic issue.

### Related Issues/PRs
If you know of any related issues or pull requests, link them here.

### Proposed Solution (optional)
If you have any ideas on how to fix the bug or potential areas to investigate, suggest them here.

### Your Contribution (optional)
Update the linkedin profile link with the correct one

### LINKED ISSUE #371: Add feedback for mentor Adriana
### Task
Add mentee feedback for the mentor - Adriana

**Mentee** - Nevena Verbič
**Mentor** - Adriana Zencke Zimmermann
**Type** - Both
**Feedback** - 
Adriana Zencke Zimmermann is my long-term mentor in the Women Coding Community.

From our first session, Adriana left an immensely positive impression on me. With an honest and compassionate approach to a person, she generously shares her knowledge and experiences, she is positively oriented with a problem solving attitude, she listens attentively and shows genuine interest in my life situation and aspirations. I have also immediately noticed her exceptional organizational skills, for which I admired her even more.

During the mentorship, Adriana has helped me in so many ways, improving my  technical skills as well as my soft skills, including communication, organization, and presentation. She has helped me gain the confidence I need and has encouraged me to pursue my goals by introducing just the right amount of new tasks and ideas. We also always have time to share life stories and support each other with ideas for improving work-life balance.
Through Adriana’s mentorship, I have strengthened my expertise in Java, Spring Boot, and API design, which feels fantastic. She guides me through tasks, helps me understand complex parts, and enables me to advance more quickly. I am grateful that she has invited me to collaborate with the WCC team, implementing a new community web platform. Collaborating with others, attending stand-up meetings, and engaging in pair programming have greatly boosted my confidence. My hope is to contribute to future mentorship efforts and help other newcomers grow within the community, just as Adriana has done for me.

As I have got to know Adriana, I have seen how much heart she puts into the community and people in general. She sees the best in us, she is eager to help, to solve problems and moves us forward. I respect her deeply, and during our conversations, I am always motivated to keep moving forward, take action, and see myself in a positive light.

The community of wonderful people who generously volunteer their time to help others enter or return to the tech industry, gain knowledge they need, boost their self-esteem and find jobs is like a magic itself and a testament to the greatness of human beings.

Thank you Adriana and Women Coding Community

**Rating** - 5
**Date** - 10/25/2024

### Acceptance criteria
Screenshot of the mentees' feedback

### PR DESCRIPTION
**Pull request by khairahscorner**
## Description
- Added Joana to WCC team
- fixed Adriana's linkedin
- added review for Adriana

## Change Type
- [ ] Bug Fix
- [ ] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [x] Data Update
- [ ] Documentation
- [ ] Other

## Related Issue
resolves #369 resolves #371 

## Screenshots
Joana on team:
Screenshot 2024-11-02 at 12 26 54 PM
Screenshot 2024-11-02 at 12 39 20 PM
Screenshot 2024-11-02 at 12 39 37 PM

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -135,4 +135,12 @@
   image: assets/images/team/14.jpeg
   network:
     - name: linkedin
-      link: https://www.linkedin.com/in/sonali-goel-6b611522/
\ No newline at end of file
+      link: https://www.linkedin.com/in/sonali-goel-6b611522/
+
+- name: Joana Brito
```
Comment (dricazenck) [inline _data/team.yml:140]: We have 2 new Leads, could you update in another PR @khairahscorner  please?

**Review by dricazenck — APPROVED** [review]
🚀

**Comment by dricazenck** [comment]
We have 2 new Leads, could you update in another PR @khairahscorner please?

Also could you double check with @joanaBrit if she does not want to include her github?

**Comment by joanaBrit** [comment]
Yes I would like. Here: https://github.com/joanaBrit, thank you!

**Pull request merged**